string id
sensor_msgs/Image imageBGR
---
facenet_ros/VisionFaceObjects faces
